### PR TITLE
[docs] 1.8.x improvemets

### PR DESCRIPTION
**Pull request by iamveritas**
add correct link to the eosjs documentation page
and made improvements based on marketing review feedback.

### CODE REVIEW

**Review by lparisc — DISMISSED** [review]
See suggestions in PR #1165

File: docs/05_features/10_resource_payer.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Find below one example that illustrate how to send transaction to the blockchain
 
 ### EOSJS
 
-To send a transaction to the blockchain using javascript via the `eosjs` library, follow the [How To Send A Transaction](https://developers.eos.io/manuals/eos/latest/cleos/how-to-guides/how-to-submit-a-transaction) steps and note in the json example the resource payer information set in the ``transaction_extensions`` field.
+To send a transaction to the blockchain using javascript via the `eosjs` library, follow the [How To Set A Payer](https://developers.eos.io/manuals/eosjs/latest/how-to-guides/how-to-set-a-payer) steps and note in the json example the resource payer information set in the ``resource_payer`` object.
```
Comment (bobgt) [inline docs/05_features/10_resource_payer.md:41]: I got paused at the "note in the json example" - for a moment I thought that there is a "note" in the JSON. But reading it twice I understood that attention is being pointed out at the properties of the `resource_payer`` object.

Will this revision make it clearer in one stroke? 

"To send a transaction to the blockchain using JavaScript via the `eosjs` library, follow the How To Set A Payer steps. In the given JSON example, the resource payer properties are set in the `resource_payer` object."

Ignore if it just my eyes seeing it differently in first go.

**Review by bobgt — DISMISSED** [review]
Approving with a comment.

File: docs/05_features/10_resource_payer.md (hunk 1 above)
Comment (iamveritas) [reply docs/05_features/10_resource_payer.md]: follow [...] and note
two verbs

Comment (iamveritas) [reply docs/05_features/10_resource_payer.md]: I re-worded it.

**Review by bobgt — APPROVED** [review]

**Review by lparisc — APPROVED** [review]

**Pull request merged**